modules/items/server/models/item.server.model.js

```diff
@@ -51,7 +51,12 @@ var ItemSchema = new Schema({
   modules: {
     type: Schema.ObjectId,
     ref: 'Module'
-  }
+  },
+  pdf: {
+     type: String,
```

Comment:
String should be fine for this, I don't really know if there's a url type in mongo

Reply:
a quick google search says no